config/jobs/kubernetes/sig-cloud-provider/azure/sig-azure-config.yaml

```diff
@@ -269,3 +269,40 @@ periodics:
     testgrid-alert-email: kubernetes-provider-azure@googlegroups.com
     description: Runs conformance & node conformance tests latest kubernetes master with aks-engine
     testgrid-num-columns-recent: '30'
+- interval: 24h
+  name: ci-kubernetes-e2e-capz-conformance-master
+  decorate: true
```

Comment:
The default timeout is 2h. If the job takes around 2h, you should try increasing the timeout to prevent accidental timeout before the job is finished.
```
decoration_config:
  timeout: 3h
```